### PR TITLE
SQL-1323: Autogenerate UUID upgrade code to avoid bump commits

### PR DESCRIPTION
**Pull request by pmeredit**

### CODE REVIEW

File: evergreen.yml [hunk 1]
```diff
@@ -235,7 +235,14 @@ functions:
...
           cd installer/msi
-          VERSION="$RELEASE_VERSION"
+          if [ "$RELEASE_VERSION" == "snapshot" ]; then
+              MINOR_VERSION="0.1"
+              VERSION_LABEL="0.1.0"
+          else
+              MINOR_VERSION=$(echo "$RELEASE_VERSION" | sed 's|\([0-9]\+[.][0-9]\+\)[.][0-9]\+|\1|')
+              VERSION_LABEL="$RELEASE_VERSION"
+          fi
+          UUID=$(uuidgen -m -N "mongoodbc_no_conflicts_$MINOR_VERSION" -n @oid)
```
Comment (nbagnard) [inline evergreen.yml:245]: [nit] This is probably where I'd add a comment.
Something like "Retrieve the unique UID for the minor version. This allows us to automatically flag releases with breaking changes (minor or major version changed) and not override the existing driver during the installation process"

**Review by nbagnard — APPROVED** [review]
LGTM!

File: evergreen.yml (hunk 1 above)
Comment (pmeredit) [reply evergreen.yml:245]: sgtm

**Comment by pmeredit** [comment]
evergreen merge

**Pull request merged**